src/styles/main.css

```diff
@@ -0,0 +1,188 @@
+body {
+  margin: 0;
+  font-family: Arial, Baghdad, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 720px;
+  gap: 1px;
+  margin: 50px auto;
+}
+.calendar--month-length--28 .day:nth-child(n+29) {
+  display: none;
+}
+.calendar--month-length--29 .day:nth-child(n+30) {
+  display: none;
+}
+.calendar--month-length--30 .day:nth-child(n+31) {
+  display: none;
+}
+.calendar--month-length--31 .day:nth-child(n+32) {
+  display: none;
+}
+.calendar__start-day--mon > :first-child {
+  margin-left: 0px;
+}
+.calendar__start-day--tue > :first-child {
+  margin-left: 101px;
+}
+.calendar__start-day--wed > :first-child {
+  margin-left: 202px;
+}
+.calendar__start-day--thu > :first-child {
+  margin-left: 303px;
+}
+.calendar__start-day--fri > :first-child {
+  margin-left: 404px;
+}
+.calendar__start-day--sat > :first-child {
+  margin-left: 505px;
+}
+.calendar__start-day--sun > :first-child {
+  margin-left: 606px;
+}
+
+.day {
```

Comment:
The task specifies that the calendar should have 10px padding, which is not included in the current implementation.

Reply:
Please fix this - paddings are still missing

<img width="1662" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/1983879/97f4ff0b-4490-4416-a1ae-8a07f2d6ce8d">